packages/block-editor/src/components/block-mover/style.scss

```diff
@@ -1,5 +1,17 @@
 .block-editor-block-mover {
 
+	&.is-horizontal {
+		display: grid;
+		grid-template-columns: 1fr 1fr;
+		width: auto;
+		min-height: auto;
+
+		.block-editor-block-mover__control-drag-handle {
```

Comment:
We should not remove the drag handle on the horizontal movers. I guess horizontal drag&drop still does not work as expected but in that case ideally, it should be removed on the block level.